### PR TITLE
GMD integration

### PR DESCRIPTION
**Pull request by kavsky02**

### CODE REVIEW

File: contracts/facets/avalanche/GMDFacet.sol [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+contract GMDFacet is ReentrancyGuardKeccak, OnlyOwnerOrInsolvent{
+    // CONSTANTS
+
+    address private constant GMDVaultAddress = 0x5517c5F22177BcF7b320A2A5daF2334344eFb38C;
+
+    // PUBLIC FUNCTIONS
+
+    function gmdStakeUSDC(uint256 amountStaked, uint256 minSharesOut) public {
+        IGMDFacet.StakingDetails memory position = IGMDFacet.StakingDetails({
+            asset : 0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E,
```
Comment (mbare0) [inline contracts/facets/avalanche/GMDFacet.sol:24]: shouldn't we use `TokenManager` here?

File: contracts/interfaces/facets/avalanche/IGMDFacet.sol [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    function gmdUnstakeUSDC(uint256 amountUnstaked, uint256 minTokenOut) external;
+    function gmdUnstakeAVAX(uint256 amountUnstaked, uint256 minTokenOut) external;
+    function gmdUnstakeBTCb(uint256 amountUnstaked, uint256 minTokenOut) external;
+    function gmdUnstakeWETHe(uint256 amountUnstaked, uint256 minTokenOut) external;
+
+    struct StakingDetails {
+        address asset;
+        address receiptToken;
+        bytes32 symbol;
```
Comment (mbare0) [inline contracts/interfaces/facets/avalanche/IGMDFacet.sol:17]: what if someone adds symbol that does not correspond to the correct asset address

File: common/addresses/avax/token_addresses.json [hunk 3]
```diff
@@ -9,6 +9,10 @@
   "PNG": "0x60781C2586D68229fde47564546784ab3fACA982",
   "PTP": "0x22d4002028f537599bE9f666d1c4Fa138522f9c8",
   "JOE": "0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd",
+  "gmdUSDC": "0x33f0a866d9024d44de2E0602f4C9B94755944B6F",
+  "gmdAVAX": "0x13AF25f924056d4D4668705C33aB9b70D505050e",
+  "gmdBTC": "0x8fe3024351B9a51a3439183e940c2aF3994DD52F",
+  "gmdETH": "0xE28c95e9EB0f6D16b05D265cAa4BcEE9E5C2e625",
```
Comment (mbare0) [inline common/addresses/avax/token_addresses.json:15]: we could be more specific with names, like `gmdWETHe` - to follow the method names and take into account new vaults that might come

**Review by 0xlock — APPROVED** [review]
LGTM

File: common/addresses/avax/token_addresses.json (hunk 3 above)
Comment (kavsky02) [reply common/addresses/avax/token_addresses.json]: Done ✅

**Pull request merged**